libs/remix-ui/workspace/src/lib/remix-ui-workspace.tsx

```diff
@@ -109,6 +117,18 @@ export const Workspace = (props: WorkspaceProps) => {
     }
   }, [])
 
+  const createNewWorkspace = async (workspaceName) => {
```

Comment:
maybe it would make sense to refactor this with `onFinishCreateWorkspace` because they do the same thing.